### ISSUE TITLE
Product delivery

### ISSUE THREAD

**Issue by jamalv** [teamforus/general#853]
As a requester, I would like products to be delivered at home.

<p></p>

<p></p>

**Comment by jamalv** [comment]
Link to feature: https://forus.productboard.com/feature-board/planning/features/10533767

**Comment by maxvisser** [comment]
I am placing the label big. (not too big). Currently we have already built a new reservation system. This system currently does not allow adding address information. A concern I already would have is the ....

**Comment by jamalv** [comment]
@maxvisser we updated the user stories to show what's 'must have' does this impact the estimation?

**Comment by maxvisser** [comment]
As a requester, I would like to fill in my address when delivering a product. 
scope: small

As a provider, I would like to see the address of the requester so that I can send the product to the home of the requester
Scope: small

These two would be scope medium 


but I suspect users don't want to fill in again and again their NAW-data; so we should probably aswell store this data?:
As a requester I want to save my NAW-data for next reservation
Scope: Small

I would also suspect the provider don't want to ship their product everywhere and would want the municipality or citizen to pay for shipping. 
As a provider I want the requester or sponsor to pay for shipping so I can sent the product to their home
Scope: Medium

Aswell I think the provider would want to select which product can be ordered and which can't:
As a provider I want to select pickup location (online / physical) so I can tell the citizen if it will be shipped to their home.
Scope: Small

**Comment by maxvisser** [comment]
So I would estimate it on big / too big; I would start with allowing saving NAW-data / pickup location before starting on ordering and a research into shipping costs.

**Comment by linda-90** [comment]
can we break it into phases?

**Comment by maxvisser** [comment]
Yes I would suggest starting with the data (could maybe be part of our research into BRP / Haalcentraal api) and a research into shipping costs (what is desired, that user pays this from their voucher or that this is directly payed from the fund? or is iDEAL required?)

**Comment by linda-90** [comment]
Yes, a research into shipping costs sounds good. Let's start small: Pay from voucher credit. As an extra, it would of course be nice to pay with ideal.
